mltsp/Flask/static/js/scripts.js

```diff
@@ -1828,3 +1822,50 @@ function drawScatterplotMatrix(datafilename){
     });
 
 }
+
+
+
+function generate_pred_results_table(div_id, results_dict){
```

Comment:
@stefanv, as far as I know that would require a page reload, which is not what's happening here.